### PR TITLE
XW-203 VK tag support

### PR DESCRIPTION
**Pull request by bognix**
**Note**
This PR contains changes from XW-217 branch. Please review only commits with XW-203 prefix

### CODE REVIEW

File: front/scripts/main/components/WidgetVKComponent.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		};
+	};
+}
+
+App.WidgetVKComponent = Em.Component.extend(App.WidgetScriptStateMixin, {
+	classNames: ['widget-vk'],
+
+	elementId: null,
+	options: null,
+	groupId: null,
```
Comment (rogatty) [inline front/scripts/main/components/WidgetVKComponent.ts]: Only `data` is passed to this component.

File: front/scripts/main/components/WidgetVKComponent.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+				this.set('scriptLoaded.vk', true);
+			});
+		}
+	},
+
+	createWidget(): void {
+		if (this.get('scriptLoaded.vk')) {
+			var elementId = this.get('elementId');
+			var groupId = this.get('data.groupId');
+			var data = this.get('data');
```
Comment (rogatty) [inline front/scripts/main/components/WidgetVKComponent.ts]: Use single `var` per scope.

File: front/scripts/main/components/WidgetVKComponent.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		Widgets?: {
+			Group: Function;
+		};
+	};
+}
+
+App.WidgetVKComponent = Em.Component.extend(App.WidgetScriptStateMixin, {
+	classNames: ['widget-vk'],
+
+	elementId: null,
```
Comment (rogatty) [inline front/scripts/main/components/WidgetVKComponent.ts]: I don't think we should do that, not sure if there are any side effects but surely there are no gains.
http://emberjs.com/api/classes/Ember.Component.html#property_elementId

File: front/scripts/main/components/WidgetVKComponent.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+				this.set('scriptLoaded.vk', true);
+			});
+		}
+	},
+
+	createWidget(): void {
+		if (this.get('scriptLoaded.vk')) {
+			var elementId = this.get('elementId'),
+				data = this.get('data'),
+				groupId = this.get('data.groupId');
```
Comment (rogatty) [inline front/scripts/main/components/WidgetVKComponent.ts]: `groupId = data.groupId` ;) Or maybe it doesn't even make sense to use a variable?

**Comment by vforge** [comment]
LGTM

**Comment by rogatty** [comment]
Minor thing and LGTM. Make sure no one merges it before https://github.com/Wikia/mercury/pull/1347

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**